opentreemap/treemap/lib/map_feature.py

```diff
@@ -315,8 +315,8 @@ def context_dict_for_resource(request, resource, **kwargs):
         __, display_area = get_display_value(instance,
                                              'greenInfrastructure', 'area',
                                              resource.calculate_area())
-        display_units = get_unit_name(get_units(instance,
-                                      'greenInfrastructure', 'area'))
+        display_units = get_unit_abbreviation(
```

Comment:
Well we _do_ use full unit names in consistently and purposefully [in the audit system](https://github.com/OpenTreeMap/otm-core/blob/d586ae00dcba28c1f11226c483be29bc5f0fced8/opentreemap/treemap/audit.py#L1318-L1342) for displaying items in the plot detail sidebar and on the edits pages.

So we haven't been using abbreviations _everywhere_...

That said, going back to abbreviations fixes the display issue with diameter/circumference, and it matches what we were doing before.  It would be nice to be consistent though, and use either full names or abbreviations everywhere on the public-facing portion of the site.